01-js-intro-and-jquery/10/js/scripts.js

```diff
@@ -0,0 +1,49 @@
+$(document).ready(function() {
+	
+	var alias = '';
+	var name  = '';
+	var nameLength = '';
+	var lastChange = '';
+	
+	var focusAlias = function() {	
+		$('.alias').focus(); 
+	};
+
+	var highlightName = function() {
+		name = $('#response').text().substring(8);
```

Comment:
This is an issue for this case, you MUST store the jQuery object into a variable, if not you'll be doing a DOM search 3 times for this function, for this case will not be an issue but imagine if you got 200.000 jQuery DOM searchs and all of them are not cached up into a variable the performance goes down a lot. Try to be aware for this the next time, no need to change this :)